.babelrc

```diff
@@ -1,7 +1,6 @@
 {
   "presets": [
-    ["es2015", { "modules": false }],
+    "es2015",
```

Comment:
Why was `modules:false` removed, by the way? It was implemented in PR #8 to remove CommonJS modules and enable tree shaking.